cpp/src/arrow/compute/kernels/scalar_arithmetic_test.cc

```diff
@@ -39,7 +40,7 @@ namespace arrow {
 namespace compute {
 
 template <typename ArrowType>
-class TestBinaryArithmetics : public TestBase {
+class TestBinaryArithmetic : public TestBase {
```

Comment:
"arithmetics" is not correct, so many of the changes in this file are just this rename